lib/tasks/import_prebakedstats.rake

```diff
@@ -83,7 +87,9 @@ namespace :import do
       geo_entity = GeoEntity.find_by(name: "Disputed")
     else
       geo_entity = GeoEntity.find_by(iso3: name) || GeoEntity.find_by(name: name)
+      return unless geo_entity
```

Comment:
What's the difference between returning nothing here and waiting and then returning nil in the line below?

Reply:
Yeah you might be right in that, because if the second half of the OR condition returns `nil`, then `geo_entity` will be `nil` anyway.